xpk/xpk.py

```diff
@@ -207,10 +210,10 @@
 apiVersion: scheduling.k8s.io/v1
 kind: PriorityClass
 metadata:
-  name: veryhigh
```

Comment:
Why is very-high and verylow different? Otherwise looks great.

Reply:
great catch thank you.